front/components/featuredBrands/featuredBrands.scss

```diff
@@ -1,7 +1,6 @@
 .landing{
```

Comment:
This class should not be here as it is saying that featured HAS to be in landing.  You don't want to tightly couple your directives with anything